dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/constraints/DatadogDataConstraints.kt

```diff
@@ -16,7 +17,9 @@ import java.util.Locale
  *
  * @param internalLogger Internal logger.
  */
-class DatadogDataConstraints(private val internalLogger: InternalLogger) : DataConstraints {
+class DatadogDataConstraints(
+    private val internalLogger: InternalLogger = NoOpInternalLogger()
```

Comment:
maybe it is better to revert this change and just explicitly provide `NoOpInternalLogger` at the call site in the `benchmark` module?

Reply:
`InernalLogger` and `NoOpInternalLogger` are now inside `core`, `internal` doesn't have dependency on it, we need to migrate them first if we want to do that, what do you prefer?

Reply:
Yes, sorry, I was confused by myself, ignore what I said above. it's actually the problem of `internal` modifier on `NoOpInternalLogger`

Reply:
this is removed since Marius said sanitize is not necessary here.